internal/operators/lvm/manifest.go

```diff
@@ -110,4 +110,8 @@ metadata:
 spec:
   storage:
     deviceClasses:
-    - name: vg1`
+    - name: vg1
+	  thinPoolConfig:
+		name: thin-pool-1
+		sizePercent: 90
+		overprovisionRatio: 10`
```

Comment:
Where do these values come from?

Reply:
sizePercent = 90 is the recommended default value for the field in LVMO. This states that the thinpool will use 90% of the volumegroup.
overprovisionRatio can be set to a smaller value if required.This is basically how much the storage can be overprovisioned - if the value is set to 10 and the thinpool is 10G, you can create PVCs with sizes totaling 100G .  The admin is responsible for monitoring the actual usage to ensure that all the space in the thinpool is not used up.

A smaller value may be better here. Maybe 2.

Reply:
OverprovisionRatio is a multiplier, not a percentage.

Reply:
gotcha, I think we can ever go lower than 2
btw -> could it be that thinPoolConfig should not have been strictly required? could users opt-out somehow?

Reply:
> gotcha, I think we can ever go lower than 2 btw -> could it be that thinPoolConfig should not have been strictly required? could users opt-out somehow?

Afraid not. The thinpoolconfig is mandatory for odf lvmo.